src/app/server/Server.cpp

```diff
@@ -605,3 +708,83 @@ AdminPairingTable & GetGlobalAdminPairingTable()
 {
     return gAdminPairings;
 }
+
+#if CHIP_DEVICE_CONFIG_ENABLE_BOTH_COMMISSIONER_AND_COMMISSIONEE
+
+CHIP_ERROR InitCommissioner()
```

Comment:
The fact we're now instantiating a controller / commissioner inside the shared server code makes me feel like we're getting pretty wound around the axle here.

Why does this need to be in `Server.cpp`?  I understand that we need to run this code in many of the example accessories.  But this just does not seem like the right place to put it.